### ISSUE TITLE
Errors for missing ending brace no longer include last line

### ISSUE THREAD

**Issue by drewwells** [sass/libsass#1506]
Input:
```
div {
  height: 10px;
```

Expected error:
```
Error > stdin:2
Invalid CSS after "...  height: 10px;": expected "}", was ""
div {
  height: 10px;
```

Actual error:
```
Error > stdin:2
		Invalid CSS after "": expected "}", was ""
		div {
		  height: 10px;
```

**Comment by xzyfer** [comment]
I expect this is related to https://github.com/sass/libsass/pull/1496

**Comment by drewwells** [comment]
Confirmed it is, reverting https://github.com/mgreter/libsass/commit/139ca8d3ab3a763d905f582916445b1a34dbe91e fixes this issue

**Comment by mgreter** [comment]
This should be fixed by #1512
